src/main/kotlin/com/spectralogic/bp/bench/cli/GetFromTapeCommand.kt

```diff
@@ -3,13 +3,47 @@
  *   Copyright 2014-2019 Spectra Logic Corporation. All Rights Reserved.
  * ***************************************************************************
  */
+/*
+ * ****************************************************************************
+ *   Copyright 2014-2019 Spectra Logic Corporation. All Rights Reserved.
+ * ***************************************************************************
+ */
  
 package com.spectralogic.bp.bench.cli
 
 import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.prompt
+import com.github.ajalt.clikt.parameters.options.validate
+import com.spectralogic.ds3client.Ds3ClientBuilder
+import com.spectralogic.ds3client.helpers.Ds3ClientHelpers
+import com.spectralogic.ds3client.models.common.Credentials
+
+class GetFromTapeCommand: CliktCommand(name = "get") {
+    private val endpoint by option(
+        "-bp",
+        "--blackpearl"
+    ).prompt("Black Pearl data path").validate { require(it.isNotEmpty()) { "Black Pearl data path cannot be empty" } }
```

Comment:
What's the data type on this?  String, URI?

Reply:
The SDK client takes a string so the type here is string, but I could add more validation.